ds-prep-python-tuples/ds-prep-python-tuples/main.py

```diff
@@ -0,0 +1,51 @@
+# 3
+passenger = tuple((12, True, 'Bonnell, Miss. Elizabeth', 'female', 58))
+
+# 4
+print(passenger)
+
+# 6
+name = passenger[2]
+
+# 7
+age = passenger[-1]
+
+# 8
+print('Full Name: {}       Age: {}'.format(name, age))
+
+# 9
+survived_and_name = passenger[1:3]
+
+#10
+gender_and_age = passenger[3:]
```

Comment:
While this does technically get the last two elements of the tuple since `passenger` has exactly `5` values, a more robust way to do the same thing is to use a negative index:

---

![image](https://user-images.githubusercontent.com/16143771/148817271-ac3cffc7-8a3f-4975-9f79-e6d51463cb4c.png)